### PR TITLE
[pigeon] Added test and fixed bug where running without arguments would crash.

### PR DESCRIPTION
**Pull request by gaaclarke**

### CODE REVIEW

File: packages/pigeon/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.1.9
```
Comment (jmagman) [inline packages/pigeon/CHANGELOG.md:1]: Should this be 0.1.9+1? I think that's the convention for plugins for a patch version bump when < 0.x.x

**Review by jmagman — APPROVED** [review]
LGTM with version question.

File: packages/pigeon/CHANGELOG.md (hunk 1 above)
Comment (gaaclarke) [reply packages/pigeon/CHANGELOG.md:1]: The +X is used for changes that change the repository, but don't change the binary in anyway, like updating a README.md.

Comment (jmagman) [reply packages/pigeon/CHANGELOG.md:1]: That's what I thought, but I don't think that's how the ecosystem team does it when it's < 1.0
https://github.com/flutter/plugins/pull/2090#discussion_r328265368

**Comment by creativecreatorormaybenot** [comment]
Did not fix it for me: flutter/flutter#67227

**Pull request merged**